playground/app.js

```diff
@@ -156,20 +142,11 @@ class MongoService {
 
 const mongoservice = new MongoService();
 const mongoBackendPlugin = servicePluginFactory(mongoservice);
-const initialPipeline = [{
-  name: 'domain',
-  domain: 'test-collection',
-},
-{
-  name: 'filter',
-  condition: {
-    and: [{
-      column: 'Groups',
-      value: '<%= groupname %>',
-      operator: 'eq'
-    }]
+const initialPipeline = [
```

Comment:
I think this domain was used to test the scroll itself but that it should be commited. Am I right?

Reply:
Ah yes my bad, I changed it for testing purpose, I'll modify it